.github/workflows/tests.yaml

```diff
@@ -13,12 +13,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [2.7, 3.6, 3.7, 3.8, 3.9, "3.10-dev"]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
Seeing as this no longer has suffix does it need the double quotes? Guessing doesn't matter - but worth mentioning.

Reply:
Couldn't remember about this, so I tried omitting the quotes on a new branch, but looks like Github Actions considers it 3.1 for some reason!
> Version 3.1 with arch x64 not found The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

From https://github.com/getyoti/yoti-python-sdk/actions/runs/2509269281

Reply:
Right. OK. Great.